test/evergreen.yml

```diff
@@ -1156,8 +1183,11 @@ functions:
         set -o errexit
         set -o verbose
         ${PREPARE_TEST_ENV}
-
-        ../../../test/evergreen/checkpoint_test_predictable.sh ${times} ${checkpoint_args}
+        disagg_args=""
+        if [[ "${disagg_run|false}" == "true" ]]; then
+            disagg_args="-d leader -t row -x"
+        fi
+        ../../../test/evergreen/checkpoint_test_predictable.sh ${times} "${checkpoint_args} $disagg_args"
```

Comment:
if this runs then it's wonderful, but last I heard this (predictable replay) needed some work to be done to make it work. And we were planning on enabling these then?